src/frontend/magnify/src/components/design-system/WaitingRow/WaitingRow.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Card } from 'grommet';
+import styled, { keyframes } from 'styled-components';
+import { defineMessages, useIntl } from 'react-intl';
+
+export interface WaitingRowProps {
+  /**
+   * The background color of the card
+   */
+  background?: string;
+  /**
+   * The min height of the card
+   */
+  minHeight?: string;
```

Comment:
I find those comments useless, as they are not giving more information than the variable names.